### PR TITLE
DEMO-1 Remove irrelevant line from README

### PR DESCRIPTION
**Pull request by sertansenturk**
I pushed a bad commit to the README file in main earlier. This PR fixes that

TEST: N/A

See DEMO-1 for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,2 @@
 # git_demo
 Demo to show capabilities of git and (a bit) Github
-
```
Comment (chrispla) [inline README.md:3]: Ok I see the issue.

**Review by chrispla — CHANGES_REQUESTED** [review]
Please be mindful of whitespaces

File: README.md (hunk 1 above)
Comment (sertansenturk) [reply README.md:3]: Yep my bad.

**Review by chrispla — APPROVED** [review]
This looks clean.

**Comment by sertansenturk** [comment]
:shipit:

**Pull request merged**